pycheribuild/files/cheribsd/fstab-zfs.in

```diff
@@ -0,0 +1,5 @@
+tmpfs /tmp tmpfs rw,failok 0 0
```

Comment:
Hm, it'd be nice to avoid this duplication, I went to great lengths to get down to only one fstab... probably means splitting up the template into multiple fragments (fstab-ufs.in, fstab-swap.in and fstab-core.in), which is probably out of scope for this change, but worth an XXX comment somewhere

Reply:
I've switched to a mixed model where we have the template for constant things or things the cheribuild user might want to change (the smb mount lines), but we insert lines for things that are a function of the disk image layout.